components/shell/MobileMenu.tsx

```diff
@@ -119,9 +119,6 @@ export function MobileMenu({ toggleTheme }: { toggleTheme: () => void }) {
                   <nav className="flex h-full flex-col">
                     <div className="space-y-1">
                       {NAV_ITEMS.map((item) => {
-                        if (item.serviceAccountOnly && !isServiceAccount) {
-                          return
-                        }
                         const Icon = ICONS[item.icon]
```

Comment:
Does this end up showing the "Configure" nav item to non-service accounts?  The main issue there is that only the service account has the ability to modify their service record, so it wont have the intended effect if another user does it.

Reply:
(There's also similar logic in SidebarNav.tsx)

Reply:
oh this wasn't meant to be committed. cleaned up.